### PR TITLE
bootstrap: install open-vm-tools binaries from RPMs

### PR DESCRIPTION
**Pull request by vrutkovs**
This ensures vmtoolsd would be started on bootstrap nodes for vSphere installs.

### CODE REVIEW

**Comment by vrutkovs** [comment]
/retest

flakes

File: bootstrap/overlay/etc/systemd/system/multi-user.target.wants/vmtoolsd.service [hunk 1]
```diff
@@ -0,0 +1 @@
+../vmtoolsd.service
```
Comment (LorbusChris) [inline bootstrap/overlay/etc/systemd/system/multi-user.target.wants/vmtoolsd.service:1]: @vrutkovs does this service already run only conditionally on vmsphere?

Comment (vrutkovs) [reply bootstrap/overlay/etc/systemd/system/multi-user.target.wants/vmtoolsd.service:1]: I don't think so - the service is provided from RPM and effectively a noop on other platforms. Also it doesn't fail so its safe to have it running everywhere.

I guess we could add a dropin to run only on vsphere later

**Pull request merged**